bootstrapvz/common/tasks/ssh.py

```diff
@@ -109,4 +110,8 @@ def run(cls, info):
         public = [path + '.pub' for path in private]
 
         from ..tools import log_check_call
-        log_check_call(['shred', '--remove'] + private + public)
+        for ssh_host_key in private + public:
+            if os.path.isfile(ssh_host_key):
```

Comment:
Why the change? Do some of the keys not exist? In that case they should be removed from the list, we should always strive to know what *exactly* is on the image, so that we can fail if something is missing unexpectedly.

Also, `ssh_host_keys` should probably be distinguished form the other variable with something more than just an underscore. Alternatively, simply negate the `isfile()` and remove the key from the list.